community-operators/hazelcast-enterprise/hazelcast-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,207 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    alm-examples: |-
+      [{
+       "apiVersion": "hazelcast.com/v1alpha1",
+       "kind": "Hazelcast",
+       "metadata": {
+          "name": "hz"
+       },
+       "spec": {
+          "image": {
+            "tag": "3.11.2"
+          },
+          "hazelcast": {
+            "licenseKeySecretName": "hz-license-key-secret"
+          },
+          "serviceAccount": {
+            "create": false,
+            "name": "hazelcast"
+          }
+       }
+      }]
+    categories: "Database"
+    certified: "false"
+    containerImage: hazelcast/hazelcast-enterprise-operator:0.0.1
+    createdAt: 2019-03-14T09:58:52Z
+    description: Install Hazelcast Enterprise cluster.
+    support: Hazelcast, Inc
+  name: hazelcast-operator.v0.0.1
+  namespace: placeholder
+spec:
+  customresourcedefinitions:
+    owned:
+    - description: Hazelcast Enterprise cluster.
+      displayName: Hazelcast Enterprise
+      group: hazelcast.com
+      kind: Hazelcast
+      name: hazelcasts.hazelcast.com
+      version: v1alpha1
+  description: |
+    Hazelcast IMDG Enterprise is the most widely used in-memory data grid with hundreds of thousands of installed clusters around the world. It offers caching solutions ensuring that data is in the right place when it’s needed for optimal performance.
+
+    ### Before Your Start
+
+    Add Security Context Constraint `anyuid` to the `hazelcast` ServiceAccount.
```

Comment:
I think this can be added to the CSV for the OpenShift version.

Reply:
https://github.com/operator-framework/community-operators/blob/master/community-operators/oneagent/dynatrace-monitoring.v0.2.0.clusterserviceversion.yaml#L203
You can use the above clusterPermissions as a template

Reply:
Good idea. Thanks for the hint. Changed.